src/exercises/exerciseTypes/Exercise.sc.js

```diff
@@ -51,6 +56,189 @@ const Exercise = styled.div`
     font-weight: 800;
   }
 
+  .headlineOrderWords {
```

Comment:
this file is becoming much too big. 
do you think you can extract the styles for the WO exercise in it's own `ExerciseTypeWO.sc.js` maybe? that would be awesome!

Reply:
Sure thing, I will revert the changes, I have created a ExerciseTypeWO.sc.js in the same folder as the OrderWords code.